src/jobs/aws_deploy.yml

```diff
@@ -9,20 +9,32 @@ parameters:
   aws_profile:
     type: string
     description: "The AWS profile to be used"
-  deployment_type:
-    type: string
-    description: "The AWS deployment type"
-  cluster_name:
-    type: string
-    description: "The Cluster name to be used"
+    default: "default"
   region:
     type: string
     description: "The aws region for the deployment"
+    default: $(python -c 'import boto3;print(boto3.Session().region_name)')
```

Comment:
I will leave this as a plain string, to not rely on having python available

Reply:
The problem here is that it will override whatever is in the credentials file.
But you are right, I don't think this will work anyway.

Reply:
```suggestion
    default: eu-west-1
```